src/main/java/com/example/sample/application/JakartaConcurrencyService.java

```diff
@@ -18,27 +18,26 @@
 import javax.naming.InitialContext;
 import javax.naming.NamingException;
 
+import jakarta.annotation.Resource;
 import jakarta.enterprise.concurrent.ContextService;
 import jakarta.enterprise.concurrent.ManagedExecutorService;
 import jakarta.enterprise.concurrent.ManagedThreadFactory;
 import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.inject.Inject;
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.core.MediaType;
 
 @Path("/api")
-@ApplicationScoped
 public class JakartaConcurrencyService {
 
-    @Inject
+    @Resource
     ContextService contextService;
 
-    @Inject
+    @Resource
     ManagedExecutorService managedExecutorService;
 
-    @Inject
```

Comment:
Switching from `@Inject` to `@Resource` since `@Inject` is a new feature coming in Jakarta Concurrency 3.1. Since nothing else in the sample is using CDI, with this change CDI can be removed all-together.